app/controllers/collections_controller.rb

```diff
@@ -80,6 +82,20 @@ def after_save(collection)
     redirect_to dashboard_path
   end
 
+  def send_depositor_notifications(collection, new_depositors)
```

Comment:
perhaps nitpicky, but does `new_depositors` makes sense here since we send to all depositors on line 33? While not a reason to hold this up, while reading this, the name made me go back and see how it was used - so a bit confusing.

Reply:
good call, i'll switch this to `depositors`